tasks/fieldStatusTracker_test.go

```diff
@@ -0,0 +1,125 @@
+package tasks
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestShouldBuildFieldMapForTask(t *testing.T) {
+	task := &RealVNCServerTask{
```

Comment:
I feel like the tests of field status tracker should not be tied to real vnc server task. I think you should create a separate struct here with appropriate fields that will be used for tests only.

Reply:
agreed. done.